bin/mesh_generator.dart

```diff
@@ -82,13 +82,13 @@ void main(List<String> args) {
     print('');
     return;
   }
-  final String type = fixedArgs.removeAt(0);
-  final GenerateFunction generator = generators[type];
+  final type = fixedArgs.removeAt(0);
+  final generator = generators[type];
```

Comment:
Case in point: I have no idea what type these are now looking at the code.

Reply:
click to navigate to `generators` definition?

*sigh* Yeah, this is a subjective things. Seattle folks generally don't like the noise. 🤷‍♂